CMakeLists.txt

```diff
@@ -436,17 +436,22 @@ if(${WITH_LTTNG})
 endif(${WITH_LTTNG})
 
 #option for Babeltrace
-option(HAVE_BABELTRACE "Babeltrace libraries are enabled" ON)
-if(${HAVE_BABELTRACE})
+option(WITH_BABELTRACE "Babeltrace libraries are enabled" ON)
+if(WITH_BABELTRACE)
+  set(HAVE_BABELTRACE ON)
```

Comment:
@tchaikov 
You were correct, it missed the actual file that was the origin of the mixup.